hardwareModules/RoFICoM/software/i2c_test/vl53l1_lidar/platform/src/vl53l1_platform.cpp

```diff
@@ -0,0 +1,299 @@
+

```

Comment:
Implementaci tohoto souboru považuji za nešťastnou (alokuje to periferie "ad-hoc") a už asi chápu, co jsi měl za problém, na který ses ptal. Celé je možné to elegantně vyřešit zavedením tzv. BSP (Board support package) a měli bychom na to zmigrovat celý FW RoFICoMu. Navrhuji vyřešit, jak se osobně potkáme.